extensions/WSLExtension/Contracts/IProcessCreator.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using WSLExtension.Models;
+
+namespace WSLExtension.Contracts;
+
+/// <summary>
+/// Abstraction used to create processes throughout the WSL extension.

```

Comment:
This is an interface, not an abstract class.  Please change the comment to say `Interface`

Reply:
updated, thanks